pipelines/mix/tasks/get_mixed.py

```diff
@@ -51,6 +66,13 @@ def _task(self):
         data_map = self._get_data()
 
         self.output_normalization_flag = True
+        self.summer_normalization = True
+
+        self.study_periods = {
+            'winter_peak' : [8],
+            'spring_light' : [3],
+            'summer_peak' : [16]
+        }
```

Comment:
Define this in the constructor and the single item array is unnecessary. Why not just make it an integer? Or a ref to a constant set up top?

Reply:
Moved to constructor, the reason for the single item is incase we want to add more later on. It might be in a config file later on, but for now that's not needed.